### ISSUE TITLE
[Arcs] Can Shaper catapult from a gate port

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#55]
https://discord.com/channels/476234833572397056/963112189802201108/1278757965155008574

Shaper
>cannot Catapult move from starports, but can Catapult move from gates
So what if the gate has a starport? can the Shaper catapult from there

and also what if the gate has a rival Gate Port?

**Comment by turmoilbyrd** [comment]
Perhaps Shaper can be errata'd to say "cannot catapult from Loyal starports". This will at least make it clear rival Gate Port don't affect them.

**Comment by turmoilbyrd** [comment]
More fundamentally this touches on the question of what the Shaper is "using" for catapult, when there can be multiple target #86

**Comment by jdyearsley** [comment]
My intent is that the Shaper can choose to use the gate—"from" should be understood as meaning "by using". I'm not sure this rises to the level of errata, but I will definitely add an FAQ.

**Comment by turmoilbyrd** [comment]
Yeah if we clarify the underlying mechanism of "using" something to catapult, then perhaps it is fine sans errata
